integreat_cms/static/src/js/analytics/hix-list.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint no-magic-numbers: ["error", { "ignore": [0, 16, 200] }] */
+window.addEventListener("load", () => {
+    const table = document.getElementById("page_hix_list") as HTMLTableElement;
+    const trigger = document.getElementById("toggle-hix-score-list-trigger");
+    const maxScrollHeight = Math.trunc((table.scrollHeight + 200) / 16);
+    table.parentElement.setAttribute("style", `max-height: ${maxScrollHeight}rem`);
```

Comment:
I guess `rem` is preferable for accessibility, since the user can change e.g. the font size to 150% on their system if they have bad eyes, and all values with `rem` will scale accordingly and all values with `px` will stay fixed?

Reply:
Good point... Though in this particular case, that would only be relevant if the user changes the font size after loading the page, I guess

But it's a moot point due to your suggestion anyways 😄

Reply:
My reasoning was mainly to stay consistent. I was taught, that while you should prefer rem instead of px, the most important thing is to stay consistent and use whatever way you choose everywhere

Reply:
That's also a good point. And our CSS framework tailwind also uses rem everywhere, so that's probably another good reason to stick with it.